theme/inc/extras.php

```diff
@@ -105,4 +105,47 @@ function braces_setup_author() {
 		$GLOBALS['authordata'] = get_userdata( $wp_query->post->post_author );
 	}
 }
-add_action( 'wp', 'braces_setup_author' );
\ No newline at end of file
+add_action( 'wp', 'braces_setup_author' );
+
+/**
+ * Modify the callback of categories / archives widgets to allow us
+ * to filter output markup
+ *
+ * @action dynamic_sidebar_params
+ */
+function braces_alter_widget_callback( $params ) {
```

Comment:
This was ported over as to add classes around the counts. I think this is a quick improvement. Although typically clients never use the core widgets, if they do, we can style them accordingly.